doc/sphinx/Pacemaker_Explained/resources.rst

```diff
@@ -159,59 +158,6 @@ The ``stonith`` standard is used for managing fencing devices, discussed later
 in :ref:`fencing`.
 
 
-.. index::
```

Comment:
Could also drop `nagios` from the `pacemakerd --features` output in Clusters from Scratch

Reply:
That's still the correct output as long as we use AlmaLinux 9 there (though the output could use updating for the latest 9)

Reply:
Unless it was built without nagios support. I was just thinking of removing that as the last mention. But it's completely unimportant.

Reply:
we use the stock AlmaLinux packages in the doc, so it's there :)